www/base/src/app/common/directives/buildsummary/buildsummarytooltip.tpl.jade

```diff
@@ -1,37 +1,46 @@
 .panel.panel-default(ng-class="results2class(buildsummary.build)", style="margin-bottom:0px")
     .panel-heading
         .flex-row(ng-if="buildsummary.build.started_at")
-            .flex-grow-1.text-left
+            .flex-grow-1
                 span(ng-if="buildsummary.prefix") {{buildsummary.prefix}}#{' '}
                 span
-                    | {{buildsummary.builder.name}}#{' '}
+                    | {{buildsummary.builder.name.slice(0,80)}}#{' '}
```

Comment:
we should probably have the ellipsis logic in the controller, or at least factor it

Reply:
I modified the string truncation and ellipsis logic in the template to be a little bit more readable.  I hope this change is acceptable.